### DISCUSSION TITLE
关于将oh my dde与deepin-tweak合并的请求。Request about merging oh-my-dde with deepin-tweak.

### DISCUSSION THREAD

**Discussion by Huayuluoshi** [linuxdeepin/developer-center#4269] [Features Request & Ideas | 特性请求 & 头脑风暴]
深度商店已经上架一款深度桌面定制工具叫"oh my dde"，已在gitee开源。其中很多功能是很棒的，希望深度定制可以直接参考。（我认为可以把oh my dde的一些设置直接搬过来，并且原作者已经因为深度定制的出现而停止更新。）
The Deepin-appstore has launched a desktop customization tool called 'oh my dde', which have been  opened  source code on Gitee. Many of these features are great, and I hope deepin-tweak can be directly referenced. (I think some of the settings for oh my dde can be directly moved over, and the original author has stopped updating due to the emergence of deepin-tweak.)

**Comment by BLumia** [comment]
不知道 oh-my-dde 的原作者有没有兴趣直接参与贡献，一起来搞。如果有的话大概可以一起完善 deepin-tweak
